src/wp-includes/class-wp-dependencies.php

```diff
@@ -490,4 +490,34 @@ public function set_group( $handle, $recursion, $group ) {
 
 		return true;
 	}
+
+	/**
+	 * Get etag header for cache validation.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param string   $wp_version WordPress version.
+	 * @param string[] $load       Array of script or style handles to load.
+	 * @return string Etag header.
+	 */
+	public function get_etag( $wp_version, $load ) {
+		$etag = "WP:{$wp_version};";
+
+		foreach ( $load as $handle ) {
+			if ( ! array_key_exists( $handle, $this->registered ) ) {
+				continue;
+			}
+
+			$ver   = $this->registered[ $handle ]->ver ? $this->registered[ $handle ]->ver : $wp_version;
```

Comment:
```suggestion
			$ver   = $this->registered[ $handle ]->ver ?? $wp_version;
```

Reply:
@vraja-pro can you updated this when you have a moment?